src/navier_stokes/INSVCStaggeredPressureBcCoef.cpp

```diff
@@ -327,9 +327,14 @@ INSVCStaggeredPressureBcCoef::setBcCoefs(Pointer<ArrayData<NDIM, double> >& acoe
                     }
                     else if (d_mu_interp_type == VC_HARMONIC_INTERP)
                     {
-                        mu = mu_contains_cells ?
-                                 2.0 * ((*mu_data)(i_g) * (*mu_data)(i_i)) / ((*mu_data)(i_g) + (*mu_data)(i_i)) :
-                                 -1.0e305;
+                        double mu_harmonic;
+                        if (IBTK::rel_equal_eps((*mu_data)(i_g), 0.0) || IBTK::rel_equal_eps((*mu_data)(i_i), 0.0))
+                            mu_harmonic = 0.0;
+                        else
+                            mu_harmonic =
+                                2.0 * ((*mu_data)(i_g) * (*mu_data)(i_i)) / ((*mu_data)(i_g) + (*mu_data)(i_i));
+
+                        mu = mu_contains_cells ? mu_harmonic : -1.0e305;
```

Comment:
What's going on here? Should this throw an error instead?

If we need to assign a value that should never be read again then we should use a signaling nan.